jest-setup.ts

```diff
@@ -1,4 +1,6 @@
 import '@testing-library/jest-dom';
+// eslint-disable-next-line import/no-unresolved
+import 'web-streams-polyfill/polyfill';
```

Comment:
I noted the commit message stated this was added for the unit tests, but would you mind expanding on why or what it addresses? Thanks!

Reply:
Sure! I had to add this polyfill because when importing `@php-wasm/universal` in a unit test, it fails due to not being able to find the `ReadableStream` class. [This class is part of the web streams/Streams API](https://developer.mozilla.org/en-US/docs/Web/API/ReadableStream).

Reply:
This makes me think that would be nice to add an inline comment here to explain why we need to import this 😅.

Reply:
Ah, OK. So `ReadableStream` is available in both Node.js and Electron's renderer, but not within the Jest environment. 👍🏻 Thank you!